docs/Contributing/Release.md

```diff
@@ -0,0 +1,143 @@
+# Release Process
+
+
+## Prerequisites:
+- Prepare release notes
+
+
+## Weekly chores
+### Enarx dependency update
+- Update local code
+  ```bash
+  git fetch --all
+  git checkout main
+  git pull --ff-only
+  ```
+- Checkout chore branch
+  ```bash
+  git checkout -b update-prerequisites
+  ```
+- Check to see `Cargo.toml` is refering to the latest updated prerequisite crates
+- Run cargo-update on all individual sub-crates in updated packages are referenced
+   ```bash
+   for d in internal/*/ ; do (cd "$d" && cargo update); done
+   ```
+- Git commit sub-crate update
+  ```bash
+   git commit -asm 'Updated Enarx sub-crate dependencies'
+   ```
+- Run cargo update on `enarx`
+  ```bash
+  cargo update
+  ```
+- Git commit sub-crate update
+  ```bash
+   git commit -asm 'Updated Enarx dependencies'
+   ```
+- Run build and tests
+  ```bash
+  cargo clean
+  cargo build
+  cargo tests
+  ```
+- Create PR
+  ```bash
+  git push origin update-prerequisites
```

Comment:
This will only work if you have access to upstream repo, normally PRs are expected to be submitted from a fork. Normally creating PR branches in the same repo is discouraged, because that way we will likely accumulate a lot of old, unused branches over time.

Reply:
Originally I was using `upstream` referring to github.com/enarx/enarx.git and `origin` referring to my fork, but per @haraldh's previous comment I tried to simplify this to use a single remote. For updating the prerequisites, I think it might be good to reuse a single `chore/cargo-update` remote branch. Thoughts?